conf/nonclangable.conf

```diff
@@ -269,6 +269,11 @@ CFLAGS:remove:pn-mozjs:toolchain-clang = "-fno-tree-vrp"
 CFLAGS:append:pn-ffmpeg:riscv64 = " -march=rv64gczbb"
 CFLAGS:append:pn-ffmpeg:riscv32 = " -march=rv32gczbb"
 
+# otherwise systemd efi fails to build
+# error: the 'sse' unit is not supported with this instruction set
+#TUNE_CCARGS:remove:pn-systemd:toolchain-clang = "-mfpmath=sse"
+#TUNE_CCARGS:remove:pn-systemd-boot:toolchain-clang = "-mfpmath=sse"
```

Comment:
This options are commented out any reasons

Reply:
Sorry, missed it! was for my last test to confirm if they are needed. Fixed